usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -305,14 +305,17 @@ def wait_for_process(process, start_time, download_job):
 
     # Let the thread run until it finishes (max MAX_VISIBILITY_TIMEOUT), with a buffer of DOWNLOAD_VISIBILITY_TIMEOUT
     sleep_count = 0
-    while process.is_alive() and (time.perf_counter() - start_time) < MAX_VISIBILITY_TIMEOUT:
+    while process.is_alive():
+        if not download_job.monthly_download and (time.perf_counter() - start_time) > MAX_VISIBILITY_TIMEOUT:
+            break
```

Comment:
What exactly was the impetus of this? Are we trying to allow the monthly download to take infinite time?

Reply:
If a download takes to long, kill it.

Unless it is a monthly download job, then disable the timeout check